Load particles from list (#506)
lethe-cfd/lethe: doc/source/parameters/resolved_cfd-dem/resolved_cfd-dem.rst

* The ``lubrication range max`` parameter defines the distance below which the lubrication force between 2 particles or between a particle and a wall is calculated. The range is defined as a multiple of the smallest cell. The lubrication force model is used to model the force between particles when they are too close to each other to accurately resolve the flow between them.

> * The ``load particles from a file`` boolean defines whether the particles are generated from a file instead of the parameter function. If this parameter is activated, the particles defined in the particle's subsection and the number of particles are ignored. The number of particles is defined directly from the file.

Reviewer: ```suggestion
* The ``load particles from file`` boolean defines whether the particles are generated from an external file instead of the prm file. If this parameter is activated, The number of particles is defined directly from the file, that is, the particle's subsection and the number of particles are ignored. 
```

> * The ``load particles from a file`` boolean that define if the particles are generated from a file instead of the parameter function. If this parameter is activated the particle defined in the particle's subsection are ignore as well as the number of particles. The number of particles is define directly from the file.

Reviewer: ```suggestion
* The ``load particles from a file`` boolean defines whether the particles are generated from a file instead of the parameter function. If this parameter is activated, the particles defined in the particle's subsection and the number of particles are ignored. The number of particles is defined directly from the file.
```

> * The ``particles file`` The file from which the particles are defined. Each line corresponds to a particle and all the relevant variables. The file must contain the following information for each particle (the header must be defined accordingly): ID p_x p_y p_z v_x v_y v_z omega_x omega_y omega_z radius density inertia pressure_x pressure_y pressure_z youngs_modulus restitution_coefficient friction_coefficient poisson_ratio rolling_friction_coefficient

Reviewer 2: Why do you keep the ID in the file if you don't use it?

> .. note::

Reviewer 2: This note needs to be moved back to the right place

> * The ``particles file`` The file from which the particles are define. Each lines correspond to a particles and all the relevant variables. The file must contain the following information for each particles and the header of the file must be defined as such: ID p_x p_y p_z v_x v_y v_z omega_x omega_y omega_z radius density inertia pressure_x pressure_y pressure_z youngs_modulus restitution_coefficient friction_coefficient poisson_ratio rolling_friction_coefficient

Reviewer: ```suggestion
* The ``particles file`` The file from which the particles are defined. Each line corresponds to a particle and all the relevant variables. The file must contain the following information for each particle (the header must be defined accordingly): ID p_x p_y p_z v_x v_y v_z omega_x omega_y omega_z radius density inertia pressure_x pressure_y pressure_z youngs_modulus restitution_coefficient friction_coefficient poisson_ratio rolling_friction_coefficient
```
